client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -504,7 +504,7 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace */ }
 				<div className="main main-column calypsoify is-iframe" role="main">
 					{ ! isIframeLoaded && <Placeholder /> }
-					{ shouldLoadIframe && (
+					{ ( shouldLoadIframe || isIframeLoaded ) && (
```

Comment:
Since `shouldLoadIframe` is only intended to handle cached nonces, I think is safe to assume that once the iframe is loaded it has been rendered with a valid nonce.